proprietary/tokens/src/components/amsterdam/grid.tokens.json

```diff
@@ -3,13 +3,20 @@
     "grid": {
       "column-count": { "value": "4" },
       "gap": {
-        "value": "clamp(1rem, calc(3.125vw + 0.375rem), 3.5rem)",
-        "comment": "Grows from 16px at 320px wide to 56px at 1600px wide."
+        "xs": { "value": "{amsterdam.space.xs}" },
+        "sm": { "value": "{amsterdam.space.sm}" },
+        "md": { "value": "{amsterdam.space.md}" },
+        "lg": { "value": "{amsterdam.space.lg}" },
+        "xl": { "value": "{amsterdam.space.xl}" }
```

Comment:
`--amsterdam-grid-gap-xs` and `xl` are never used, why are they defined?

Reply:
Yes, `Grid` only allows for a `small` or `large` gap and vertical padding. We never really tested these quick choices. Let’s keep them unchanged (although `large` is now 1.5x as large as `medium`, not 2x) and add the `extra-` options for Grid gap and padding if we need them. Removed these two.